frontend/packages/topology/src/components/ElementWrapper.tsx

```diff
@@ -10,11 +10,17 @@ type ElementWrapperProps = {
 
 // in a separate component so that changes to behaviors do not re-render children
 const ElementComponent: React.FC<ElementWrapperProps> = observer(({ element }) => {
-  const Component = React.useMemo(
-    () => element.getController().getComponent(element.getKind(), element.getType()),
-    [element],
-  );
+  const Component = React.useMemo(() => {
+    try {
+      return element.getController().getComponent(element.getKind(), element.getType());
+    } catch (e) {
+      return null;
+    }
```

Comment:
Why are we suppressing errors? This should always resolve to a component otherwise it's a dev error.

Reply:
The issue is that we create an ID on the operator group based on the application group to which it belongs. When moving a group out of an application group, the ID changes and then it cannot be found until after the backend refreshes and the group is updated. 

Creating the ID based on the grouping is the crux of the issue but I could not find a way to keep the ID consistent and unique giving that the same named operator group can be duplicated across application groups (the operator group is not data backed and relies on the part-of labels of the operator resources).

Reply:
I'm confused. How does this part have anything to do with IDs? It's only doing a lookup based on kind and type.

Reply:
Move call inside of a mobx `action` to prevent the exception from occurring. It was updating in the middle and trying to render a detached node.